db/models.py

```diff
@@ -1 +1,53 @@
 from django.db import models
+from django.db.models import CharField
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Skills(models.Model):
+    name = models.CharField(max_length=255)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="skill_set",
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class PlayerModel(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="players_race",
+        on_delete=models.CASCADE)
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue in all places